src/Aspire.Dashboard/Components/ResourcesGridColumns/UnreadLogErrorsBadge.razor.cs

```diff
@@ -1,12 +1,53 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using Aspire.Dashboard.Model;
+using Aspire.Dashboard.Otlp.Model;
+using Aspire.Dashboard.Otlp.Storage;
+using Microsoft.AspNetCore.Components;
 
 namespace Aspire.Dashboard.Components;
 
 public partial class UnreadLogErrorsBadge
 {
+    private int _unviewedCount;
+
+    [Parameter, EditorRequired]
+    public required ResourceViewModel Resource { get; set; }
+    [Parameter, EditorRequired]
+    public required Dictionary<OtlpApplication, int>? UnviewedErrorCounts { get; set; }
```

Comment:
Properties like these aren't expected to change over time IIUC. Can these be `init` properties? Ditto elsewhere in the PR.